components/eventing-controller/pkg/handlers/beb.go

```diff
@@ -16,14 +17,27 @@ import (
 	"github.com/kyma-project/kyma/components/eventing-controller/pkg/handlers/eventtype"
 )
 
+const (
+	MaxBEBSubscriptionNameLength = 50
```

Comment:
Does it make sense to make this configurable?

Reply:
That could result in different subscription names in BEB, since one could change the max length and restart the reconciler, if reconfigurable. I think we don't want that.